freqtrade/freqtradebot.py

```diff
@@ -1401,10 +1401,21 @@ def get_valid_price(self, custom_price: float, proposed_price: float) -> float:
         """
         if custom_price:
             try:
-                valid_price = float(custom_price)
+                valid_custom_price = float(custom_price)
             except ValueError:
-                valid_price = proposed_price
+                valid_custom_price = proposed_price
         else:
-            valid_price = proposed_price
+            valid_custom_price = proposed_price
+
+        cust_p_max_dist_pct = self.config.get('custom_price_max_distance_percent', 2.0)
```

Comment:
At first, i was curious to see the default set to 2.0 - assuming "why is it set to 200%" ... 

I think the way this is done (using percent instead of a ratio) will be pretty confusing and missaligned with the rest of the bot / bot settings.

All freqtrade settings (at least the ones i'm aware of) that need a % must be within a range of 0-1 (1 = 100% - which means they're a ratio - you get 100% by multiplying with 100).

This new setting is now in a range between 0 and 100 - which means 0.2 is 0.2% - not 20% (as i'd expect from having experience with all other settings.

Reply:
I will change that to a 0-1 range